mease_lab_to_nwb/convert_syntalos/syntalosimageinterface.py

```diff
@@ -36,17 +36,20 @@ def run_conversion(self, nwbfile: NWBFile, metadata: dict):
         video_folder = Path(self.source_data['folder_path'])
         video_file_path_list = [str(x.absolute()) for x in video_folder.iterdir() if x.suffix == ".mkv"]
 
-        video_timestamps = list()
+        video_timestamps = np.empty(0)
         for video_file_path in video_file_path_list:
             video_time_file = pd.read_csv(video_file_path.replace(".mkv", "_timestamps.csv"), header=0)
-            video_timestamps.append(np.array([int(x.split(";")[1]) for x in video_time_file['frame; timestamp']]))
+            video_timestamps = np.append(
+                video_timestamps,
+                [int(x.split(";")[1]) / 1E3 for x in video_time_file['frame; timestamp']]
```

Comment:
Same here:

```
video_time_df = pd.read_csv(video_file_path.replace(".mkv", "_timestamps.csv"), delimiter=';')
# note that there is an additional space in timestamps
ts_column = [col for col in video_time_df.columns if 'timestamp' in col][0]
video_timestamps = video_time_df[ts_column].to_numpy() / 1E3
```

Reply:
Even further cleaned with the use of `skipininitialspace` arg in `read_csv`